src/core-layers/hexagon-cell-layer/hexagon-cell-layer-vertex.glsl.js

```diff
@@ -59,20 +60,20 @@ void main(void) {
   float elevation = 0.0;
 
   if (extruded > 0.5) {
-    elevation = project_scale(instancePositions.z * (positions.y + 0.5) *
-      ELEVATION_SCALE * elevationScale);
+    elevation = instancePositions.z * (positions.y + 0.5) *
+      ELEVATION_SCALE * elevationScale;
   }
 
   // if ahpha == 0.0 or z < 0.0, do not render element
   float noRender = float(instanceColors.a == 0.0 || instancePositions.z < 0.0);
   float dotRadius = radius * mix(coverage, 0.0, noRender);
 
   // project center of hexagon
-  vec4 centroidPosition = vec4(project_position(instancePositions.xy), elevation, 0.0);
+  vec3 centroidPosition = vec3(instancePositions.xy, elevation);
+  vec3 offset = vec3(vec2(rotatedPositions.xz * dotRadius), 0.);
 
-  vec4 position_worldspace = centroidPosition + vec4(vec2(rotatedPositions.xz * dotRadius), 0., 1.);
-
-  gl_Position = project_to_clipspace(position_worldspace);
+  vec4 position_worldspace;
+  gl_Position = project_position_to_clipspace(centroidPosition, instancePositions64xyLow, offset, position_worldspace);
```

Comment:
Same comment as above. Just so much easier to get what is happening on this line if the first two args are similarly named.